New Pet, Frank the Turtle!

Crossedfall opened the pull request:
## About The Pull Request

Adds an adorable new pal to the mining base, Frank the Turtle (Lovingly referred to as Yeeslow by some members of the Crew). Don't let his adorable appearance fool you, Frank is built like a tank and can take a lot of punishment. Don't expect him to come to the rescue though, as he likes to get down low and hide in his shell whenever he's in danger.

**Sprite credits go to ThatGuyThere03**

![image](https://user-images.githubusercontent.com/26130695/77615933-8926ee80-6efe-11ea-9daa-db9c4619fc9b.png)![image](https://user-images.githubusercontent.com/26130695/77615963-947a1a00-6efe-11ea-95e0-a1ff270165f5.png)![image](https://user-images.githubusercontent.com/26130695/77616056-bbd0e700-6efe-11ea-9aac-a47e77bfc418.png)

![Yeescared](https://user-images.githubusercontent.com/26130695/77627570-1b39f180-6f15-11ea-8e35-f1fb2e19de21.gif)

![image](https://user-images.githubusercontent.com/26130695/77630450-1119f200-6f19-11ea-9995-6da75eccf5fd.png)


Frank Facts:

- Frank has as much heath as Bubblegum!
- Frank makes a terrible shield. When he's hiding in his shell, projectiles will fly right over him
- Butchering Frank makes you a monster, but, it will also net you a cowboy hat

## Why It's Good For The Game

He's adorable and Mining should get a pal. 

## Changelog
:cl: Crossedfall, ThatGuyThere03
add: New pet Frank (Yeeslow) added to the mining outpost
/:cl:

Crossedfall committed:
Adds yeeslow

Co-authored-by: thatguythere03 <thatguythere03@gmail.com>

Crossedfall committed:
Allows frank to hide when shot or hit.

Crossedfall committed:
Frank drops a hat when butchered

Kmc2000 commented:
IT'S PRONOUNCED YEESLOW

we now need to make him able to carry munitions to and fro

SomeAngryMiner commented:
A pet to surpass metal gear.
Now we only need him to be able to carry bags, like KMC said.

Crossedfall committed:
grammar + slightly slower speed

Even though turtles are actually kind of fast, I do like the idea of Frank being pretty slow. In this case, slower than the sloth.
